### PR TITLE
Update secure-by-default.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to configure the delivery of Simulations to users

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6613

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/secure-by-default.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ To put it another way: as a security service, we're acting on your behalf to pre
 
 You should only consider using overrides in the following scenarios:
 
-- Phishing simulations: Simulated attacks can help you identify vulnerable users before a real attack impacts your organization.
+- Phishing simulations: Simulated attacks can help you identify vulnerable users before a real attack impacts your organization. Find more information on [how to Configure the delivery of third-party phishing simulations to users and unfiltered messages to SecOps mailboxes](/security/office-365-security/configure-advanced-delivery)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/secure-by-default.md]:
```suggestion
- Phishing simulations: Simulated attacks can help you identify vulnerable users before a real attack impacts your organization. Find more information on [how to Configure the delivery of third-party phishing simulations to users and unfiltered messages to SecOps mailboxes](/security/office-365-security/configure-advanced-delivery).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge  this PR. Thanks

**Pull request merged**